CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ section for the next release.
  - Support for inspecting and plotting NetCDF output variables within the notebook workflow.
 - added support for soil temperature, relative humidity, soil moisture, and PPFD downscaling to `met_temporal_downscale.Gaussian_ensemble`
 
+- Added parallel mode for the entire SDA workflow.
+
```

Comment:
This is now in an already-released section -- Move up above line 15

Reply:
Fixed.